src/ui/server/controllers/insurance/declarations/modern-slavery/index.ts

```diff
@@ -35,7 +35,10 @@ export const pageVariables = (referenceNumber: number) => ({
   SAVE_AND_BACK_URL: `${INSURANCE_ROOT}/${referenceNumber}#`,
 });
 
-export const PAGE_CONTENT_STRINGS = PAGES.INSURANCE.DECLARATIONS.MODERN_SLAVERY;
+export const PAGE_CONTENT_STRINGS = {
+  ...PAGES.INSURANCE.DECLARATIONS.MODERN_SLAVERY,
+  EXPANDABLE: PAGES.INSURANCE.DECLARATIONS.MODERN_SLAVERY.EXPANDABLE.VERSIONS[0],
```

Comment:
TODO: Investigate if we need to update the content strings to have a top level version, instead of just `EXPANDABLE`.

Reply:
Created EMS-4061 for this.